packages/quantic/force-app/main/default/lwc/quanticSearchBox/quanticSearchBox.js

```diff
@@ -71,6 +80,10 @@ export default class QuanticSearchBox extends LightningElement {
   suggestions = [];
   /** @type {boolean} */
   hasInitializationError = false;
+  /** @type {RecentQueriesList} */
+  recentQueriesList;
+  /** @type {Array} */
```

Comment:
Do you want to defined a type above for this?

Reply:
Sure! I will add that in #4120 .